node_modules/.yarn-integrity

```diff
@@ -0,0 +1,15 @@
+{
+  "systemParams": "darwin-arm64-111",
+  "modulesFolders": [],
+  "flags": [],
+  "linkedModules": [
+    "@glimmer/syntax",
+    "bear-check-shared",
+    "glimmer-engine",
+    "prettier"
+  ],
+  "topLevelPatterns": [],
+  "lockfileEntries": {},
+  "files": [],
+  "artifacts": {}
+}
```

Comment:
What is this? 🙈

Reply:
Oops, removed! Not sure where that came from.